language/move-prover/errmapgen/src/errmap.rs

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{bail, Result};
+use move_core_types::language_storage::ModuleId;
```

Comment:
I almost wonder if we could just move this to `move-core-types`, @vgao1996 Thoughts?

Reply:
Sorry if I misunderstood, but isn't `ModuleId` already part of `move-core-types`?

Reply:
I meant move the contents of this file to `move-core-types`